seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/dag/physical/PhysicalPlanGenerator.java

```diff
@@ -393,7 +387,7 @@ private List<PhysicalVertex> getShuffleTask(
                                     TaskLocation taskLocation =
                                             new TaskLocation(
                                                     taskGroupLocation,
-                                                    taskIDPrefix,
+                                                    parallelismIndex,
```

Comment:
```suggestion
                                                    0,
```
I think `taskInGroupIndex` should be 0. Because it's only one task in it taskgroup.

Reply:
Good catch, Updated.